packages/components/src/List/Toolbar/ColumnChooserButton/service/columnChooser.utils.test.js

```diff
@@ -0,0 +1,83 @@
+import { mergedColumnsChooser, compareOrder } from './columnChooser.utils';
+
+describe('mergedColumnsChooser', () => {
+	const originalCollection = [
+		{
+			label: 'label1',
+			metadata: 'metadata1',
+		},
+		{
+			label: 'label2',
+			metadata: 'metadata2',
+		},
+		{
+			label: 'label3',
+			metadata: 'metadata3',
+		},
+	];
+	it('return a the collection merger', () => {
+		// given
+		const columnChooserCollection = [
+			{
+				label: 'label1',
+				moreStuff: true,
```

Comment:
Can you change the values between elements please ?
This test doesn't ensure that the matching by `label` is correct